airflow/models.py

```diff
@@ -667,7 +665,7 @@ def get_extra(self):
         if self._extra and self.is_extra_encrypted:
             try:
                 fernet = get_fernet()
-            except:
+            except Exception:
```

Comment:
Same here, I think it is better to raise the original exception thrown by `get_fernet()`.